extensionsGallery-insider.json

```diff
@@ -1450,7 +1450,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": "*"
+									"value": "1.29.0"
```

Comment:
This should be `>=1.29.0` (see the error in the check : https://github.com/microsoft/azuredatastudio/pull/14467/checks?check_run_id=2453507130)

Reply:
This should be done now